docs/configuration.rst

```diff
@@ -284,10 +98,10 @@ Removes a given reference set from a given data repository.
     $ ga4gh_repo remove-referenceset path/to/datarepo aReferenceSet
 
 ++++++++++++++++
-add-ontologymap
+add-ontology
```

Comment:
+1 if we are adding a true ontology, not just the mappings

Jerome Kelleher <notifications@github.com> writes:
> In docs/configuration.rst:
> 
> > @@ -284,10 +98,10 @@ Removes a given reference set from a given data repository.
> >      $ ga4gh_repo remove-referenceset path/to/datarepo aReferenceSet
> >
> >  ++++++++++++++++
> > -add-ontologymap
> > +add-ontology
> 
> Yeah, I've changed it to ontology at this level, in an attempt to future-proof
> the command a bit. See #1117.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub*
>